src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -146,7 +146,7 @@ protected final void onStartup() {
      * and including deletions only if {@link #includeDeletions()} returns {@code true}.
      */
     @Override
-    public void traverse() {
+    public final void traverse() {
```

Comment:
As discussed above, don't make this final, but do modify the comment to read: "You should only override traverse() if you are writing a new walker base class in the engine package that extends this class. It is not meant to be overridden by tools outside of the engine package."

Reply:
Fixed